modules/administration/pages/ansible-compliance-as-code.adoc

```diff
@@ -14,19 +14,43 @@ include::snippets/install_scap_security_guide_package.adoc[]
 An Ansible control node is required.
 For more information, see xref:administration:ansible-setup-control-node.adoc[].
 
-The following procedure will guide you through running remediation using an Ansible Playbook.
+The following procedure will guide you through running remediation using an Ansible playbook.
 
 .Procedure: Run Remediation Using an Ansible Playbook
+
 . From the control node system menu select menu:Ansible[Playbooks].
-  Expand the folder tab, then select a playbook.
 
-. Click the playbook.
+. In [guimenu]``Add a Playbook directory`` field, enter the path to the OpenSCAP playbooks, for example:
+
++
+
+[source,shell]
+----
+/usr/share/scap-security-guide/ansible
+----
+
+. Confirm with btn:[Save].
+
+. Navigate Go menu:Ansible[Playbooks], expand the folder you just declared, and click a playbook, for example:
```

Comment:
In 5.1, we are adding the default Playbook directory by default, so this step is not required.

In case the user has the playbook in a different directory, they can follow the link to "Setup Ansible Control Node" to find out how to add it.

It makes sense to have this step added for 5.0 and before though.